core/src/OpenACC/Kokkos_OpenACCSpace.cpp

```diff
@@ -85,8 +85,6 @@ void *Kokkos::Experimental::OpenACCSpace::impl_allocate(
 
   void *ptr = nullptr;
 
-  // FIXME_OPENACC multiple device instances are not yet supported, and thus
-  // exec_space is ignored for now.
```

Comment:
If the implementation is the same than the other overload and it is not a FIXME any more, then do
```C+
(void)exec_space;
impl_allocate(arg_label, arg_alloc_size, arg_logical_size, arg_handle);

Reply:
Updated.

Reply:
If we use `get_device_count` and `get_gpu` APIs to refer to non-host devices (e.g., GPUs) only, the current implementation will be OK, since it is valid for `get_device_count` to return 0 when the target system does not have any non-host device and the host fallback is allowed, as long as `get_gpu` is not used at all in this case.